recipe/data.go

```diff
@@ -53,3 +53,24 @@ var CPIH = Response{
 	},
 }
 
+
+var MIDYEARPOPEST = Response{
+	ID:     "40AA070E-7A43-4EC5-A1FC-84CEA2BC4461",
+	Alias:  "MIDYEARPOPEST",
+	Format: "v4",
+	InputFiles: []file{
+		{"MIDYEARPOPEST v4"},
+	},
+	OutputInstances: []instance{
+		{
+			DatasetID: "midyearpopest",
+			Editions:  []string{"time-series"},
+			Title:     "Mid Year Population Estimates",
```

Comment:
title should be `Population Estimates for UK, England and Wales, Scotland and Northern Ireland` (from the current website )